src/main/java/com/google/cloud/tools/libraries/libraries.json

```diff
@@ -1,5 +1,320 @@
+      "site":"https://cloud.google.com/speech/docs/reference/libraries#client-libraries-install-java",
+      "apireference":"https://googlecloudplatform.github.io/google-cloud-java/latest/apidocs/index.html?com/google/cloud/speech/v1beta1/package-summary.html",
+      "infotip":"The com.google.cloud.speech.v1 packages",
+      "launchStage" : "alpha",  
+      "packages" : ["com.google.cloud.speech.v1"],
+      "source" : "https://github.com/GoogleCloudPlatform/google-cloud-java/tree/master/google-cloud-speech",
+      "languageLevel" : "1.7.0",
+      "mavenCoordinates" : {
+         "groupId" : "com.google.cloud",
+         "artifactId" : "google-cloud-speech",
+         "version" : "0.23.0-alpha"
+       }
+      }
+    ]
+  },
+  {
+    "name": "Google Cloud Resource Manager",
+    "documentation": "https://cloud.google.com/resource-manager/docs/",
+    "description":"The Google Cloud Resource Manager service enables you to programmatically manage hierarchical resource containers such as organizations, folders, and projects.",
+    "transports": ["http"],
+    "clients": [
+      {
+      "name":"Java Client Libraries for the Google Cloud Resource Manager API",
+      "language":"java",
+      "site":"https://cloud.google.com/speech/docs/reference/libraries#client-libraries-install-java",
+      "apireference":"https://googlecloudplatform.github.io/google-cloud-java/latest/apidocs/?com/google/cloud/resourcemanager/package-summary.html",
+      "infotip":"The com.google.cloud.speech.resourcemanager packages",
+      "launchStage" : "alpha",  
+      "packages" : ["com.google.cloud.resourcemanager"],
+      "source" : "https://github.com/GoogleCloudPlatform/google-cloud-java/tree/master/google-cloud-resourcemanager",
+      "languageLevel" : "1.7.0",
+      "mavenCoordinates" : {
+         "groupId" : "com.google.cloud",
+         "artifactId" : "google-cloud-resourcemanager",
+         "version" : "0.23.0-alpha"
+       }
+      }
+    ]
+  },  {
+    "name": "Data Loss Prevention",
+    "documentation": "https://cloud.google.com/dlp/docs/reference/libraries",
+    "description":"The DLP API enables clients to detect the presence of Personally Identifiable Information (PII) and other privacy-sensitive data in user-supplied, unstructured data streams such as text blocks and images.",
+    "transports": ["grpc"],
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/svg/dlp.svg",
+    "clients": [
+      {
+      "name":"Java Client Libraries for the Data Loss Prevention API",
+      "language":"java",
+      "site":"https://cloud.google.com/dlp/docs/reference/libraries#client-libraries-install-java",
+      "apireference":"https://googlecloudplatform.github.io/google-cloud-java/latest/apidocs/index.html?com/google/cloud/dlp/v2beta1/package-summary.html",
+      "infotip":"The com.google.cloud.dlp.v2beta1 packages",
+      "launchStage" : "alpha",  
+      "packages" : ["com.google.cloud.dlp.v2beta1"],
```

Comment:
what does packages represent?

Reply:
The Java packages exported by the library.  We can use them in quick-fixes to add a library on an unresolved symbol.

Reply:
We have to manually manage this though. Couldn't you just build a package index from the library?

Reply:
That would require downloading each of the libraries (and possibly dependencies) and process them.  And it also doesn't help us avoid exposing shaded jars (e.g., `*.repackaged.*`).

Reply:
Aha I see how this can be useful, but I'm still skeptical of our ability to maintain this data.  Until we have concrete plans to use this data I'd prefer if we left it out.

I really like the idea of suggesting libraries based off of usage, but I'd like that feature designed first.

Reply:
This is already a feature available for plugin development in Eclipse, where Eclipse PDE scans for the packages and classes offered by the available plugins and offers to configure the plugin classpath on an unresolved symbol:

<img width="627" alt="screen shot 2017-09-18 at 4 38 08 pm" src="https://user-images.githubusercontent.com/202851/30563423-c63a466c-9c8f-11e7-8100-d4bd111063c9.PNG">

That said, the real help is PDE recognizing class names, not package names.  E.g., that `BigtableDataClient` &rarr; `com.google.cloud.bigtable`.  So +1 to removing the package names.

Reply:
@briandealwis thanks for the details and screenshot!